roles/ceph-facts/tasks/facts.yml

```diff
@@ -212,7 +212,8 @@
     ceph_uid: 167
   when:
     - containerized_deployment
-    - ceph_docker_image_tag | string is search("latest") or ceph_docker_image_tag | string is search("centos") or ceph_docker_image_tag | string is search("fedora")
+    - (ceph_docker_image_tag | string is search("latest") or ceph_docker_image_tag | string is search("centos") or ceph_docker_image_tag | string is search("fedora")
+      or (ansible_distribution == 'RedHat'))
```

Comment:
@leseb
Since we provided `v4.0.0beta-nautilus` for OSP, the `ceph_uid` fact doesn't get set because it doesn't match the list of expected tags.
I think we can assume when hosts are running a RHEL system, the containers will be RHEL as well?

Reply:
Yes, it's a fair assumption.